sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -79,37 +79,59 @@ export class ServiceBusClient {
   }
 
   /**
-   * Creates a receiver for an Azure Service Bus queue.
+   * Creates a receiver for an Azure Service Bus queue in peekLock mode.
+   *
+   * In peekLock mode, the receiver has a lock on the message for the duration specified on the
+   * queue or subscription.
```

Comment:
Since we are talking about just queues..

```suggestion
   * queue.
```

Reply:
Fixed!